packages/@sanity/studio-hints/package.json

```diff
@@ -25,10 +25,7 @@
   ],
   "dependencies": {
     "@sanity/block-content-to-react": "^2.0.7",
-    "lodash": "^4.17.15",
-    "next": "^9.1.3",
```

Comment:
I don't know what the deal is here, but `next` seems to be in use:
```js
// src/components/HintPage.js
import dynamic from 'next/dynamic'

// src/components/serializers/internalLinkSerializer.js
import Link from 'next/link'

// src/components/serializers/Video.js
import dynamic from 'next/dynamic'
```

Not sure why that is. We shouldn't be using next within studio code, obviously.
`dynamic` can be replaced with `React.lazy`, the `Link` seems to not actually be in use, so can be removed.

Reply:
Hm. Will take a look